protofsm/state_machine.go

```diff
@@ -25,6 +25,9 @@ var (
 	// ErrStateMachineShutdown occurs when trying to feed an event to a
 	// StateMachine that has been asked to Stop.
 	ErrStateMachineShutdown = fmt.Errorf("StateMachine is shutting down")
+
+	// background is a shortcut for context.Background.
+	background = context.Background()
```

Comment:
we should not do this. 

consider rebasing on top of https://github.com/lightningnetwork/lnd/pull/9342 which handles the bump to the correct `fn` version and handles updating the statemachine to thread contexts through correctly